bitrise.yml

```diff
@@ -39,6 +39,9 @@ workflows:
   _deploy_artifacts:
     steps:
     - deploy-to-bitrise-io: {}
+    - deploy-to-bitrise-io:
```

Comment:
same step twice?

Reply:
Different paths. The first is the default path that most of the steps use, and the second is where I'm putting the test output. I'll probably remove the default once I'm not using any bitrise steps.